packages/whale-api-client/src/api/poolpairs.ts

```diff
@@ -115,14 +115,24 @@ export interface PoolPairData {
     symbol: string
     displaySymbol: string
     reserve: string // BigNumber
-    blockCommission: string // BigNumber
+    blockCommission: string // BigNumber,
+    fee?: {
+      pct?: string // BigNumber
+      in?: string // BigNumber
+      out?: string // BigNumber
```

Comment:
```suggestion
      pct?: string // BigNumber
      inPct?: string // BigNumber
      outPct?: string // BigNumber
```

how about adding the Pct for all? since it looks weird without it when all or them are pct